CHANGELOG.md

```diff
@@ -1,4 +1,10 @@
-## 6.5.0 (Apr 15, 2022). Tested on Artifactory 7.37.14
+## 6.5.1 (Apr 20, 2022). Tested on Artifactory 7.37.14
+
+BUG FIXES:
+
+* provider:  Setting the right default value for 'access_token' attribute. PR: [GH-426](https://github.com/jfrog/terraform-provider-artifactory/pull/426). Issue [#425](https://github.com/jfrog/terraform-provider-artifactory/issues/425)
```

Comment:
No need to link to the issue. Just the PR.

Reply:
@chb0github requested for link to issue so added.

Reply:
You can do a relative link - you don't need the path. @alexhung I just want that so when you look at the changelog the ticket number is clickable

Reply:
Sure, I'm all for PR to be linked. Just no need to link to issue as well.

Reply:
Ok ... Leaving as is. 
ie: both PR and Issue mentioned.